src/main/scala/firrtl/passes/InferTypes.scala

```diff
@@ -12,8 +12,11 @@ object InferTypes extends Pass with PreservesAll[Transform] {
 
   override def prerequisites = Dependency(ResolveKinds) +: firrtl.stage.Forms.WorkingIR
 
+  @deprecated("This should never have been public", "1.3.2")
   type TypeMap = collection.mutable.LinkedHashMap[String, Type]
 
+  private type NTypeMap = collection.mutable.HashMap[String, Type]
```

Comment:
Why the 'N'? This holds the types for all declared components, not just nodes.

Reply:
`N` was for "new", should I rename it to make it more clear?

Reply:
I guess? I guess it doesn't really matter.

Reply:
I'll make it `New, it's 2 whole more characters and if it confused you now it'll confuse people later (myself included lol)

Reply:
I renamed it to `TypeLookup`